main/FatesHistoryInterfaceMod.F90

```diff
@@ -3393,46 +3398,58 @@ subroutine update_history_dyn2(this,nc,nsites,sites,bc_in)
 
 
                 cpatch%age_class  = get_age_class_index(cpatch%age)
+                age_class_area = sites(s)%area_by_age(cpatch%age_class)
 
                 ! Increment the fractional area in each age class bin
-                hio_area_si_age(io_si,cpatch%age_class) = hio_area_si_age(io_si,cpatch%age_class) &
+                hio_fracarea_si_age(io_si,cpatch%age_class) = hio_fracarea_si_age(io_si,cpatch%age_class) &
```

Comment:
This and the equivalent hio_fracarea_si_landuse code below can now actually be moved out of the patch loop and into an age-class loop.